### DISCUSSION TITLE
GitBook Lens is available to everyone now — powered by AI

### DISCUSSION THREAD

**Discussion by taszap** [GitbookIO/community#170] [Announcements]
Hey folks! 

[A few weeks ago](https://github.com/orgs/GitbookIO/discussions/98) we told you about our upcoming AI search feature. Well, we’re super excited to tell you that today, we’ve released it for everyone to use in an open alpha — and it’s called GitBook Lens.
Lens combines the semantic content on your GitBook pages with the power of AI to help readers find answers faster when using your documentation. [Head over to our website ](https://www.gitbook.com/solutions/ai) to find out more.

We’ve been working really hard in the last few weeks to improve the UI, and integrate Lens so it can power up both your public documentation and your internal knowledge base. We’re so proud of the results — and we hope you’ll find Lens a useful addition to your toolkit.

https://user-images.githubusercontent.com/63643069/230061491-5dc7e9aa-5a6d-4058-9080-9112150d6357.mp4

Find out how to activate and use GitBook Lens right now [in our documentation](https://docs.gitbook.com/product-tour/searching-your-content/lens).

The great news is that Lens is available to everyone while in open alpha. When Lens moves into beta, we plan to make it part of the Pro and Enterprise plans. We’ll be in touch to tell you about that before it happens, though. 

**We have a small favor to ask…**
We’ve just [launched Lens on Product Hunt](https://www.producthunt.com/posts/gitbook-ai-lens-1), and we’d love it if you could show us some support! Share some love, leave some feedback or ask a question — we’re around all day to hang out and answer. Thanks in advance!

**Give us your feedback!**
Of course, GitBook Lens is still in open alpha — so your feedback is really valuable. If you have something to share, please add a comment below, or [start a new discussion](https://github.com/orgs/GitbookIO/discussions/new?category=general&labels=pa-gitbook-ai).

Thanks so much for all your support. We hope you love GitBook Lens! 🎉

**Comment by seanthomasdevlin** [comment]
Hi, We are excited to try Lens on our public-facing docs. However, after I have enabled it, I still get no results. 
Any idea why?
Is there a wait time before the language model can digest our files and build indexes?

Thanks,
Sean

**Reply by taszap** [reply]
Hey, @seanthomasdevlin thanks for letting us know. That shouldn't be the case. 
If you are comfortable can you please share your public docs (alternatively email me at natasza@gitbook.io) and we will investigate

**Comment by seanthomasdevlin** [comment]
Thanks Natasza for the quick reply. When I came back later tonight, it was now working. So maybe it needed time to setup or you did something to fix it ;-)

Anyway, this is super super. Quite impressive! 

One feedback I have is that it would be great if it were conversational (like ChatGPT), where the user can ask follow-up questions and the bot understands the context.

Best,
Sean

**Reply by taszap** [reply]
Thanks, Sean I will share this with the team!

**Comment by JaniceManwiller** [comment]
I'm running into an odd issue with Lens. I enabled it for one of my published spaces this morning. It took a few hours, but this afternoon it finally appeared. It works fine - actually very cool.

However, a coworker is unable to see the Lens option when she goes to the same site. She has refreshed and restarted her browser, but she is still just seeing the regular search without the Lens option. She can see it on her phone.

I cannot see it from my other laptop - tried two different browsers.

Why would it not show up universally?

Thanks.

Janice

**Reply by taszap** [reply]
Hi Janice, 

Thanks for your message, it might be that your colleague still has the cached version of GitBook. 
Would you be able to please ask her to do a [hard refresh ](https://docs.gitbook.com/faq/hard-refresh#:~:text=Hold%20the%20CTRL%20key%20and%20press%20F5.,-FAQ%20%2D%20Previous)(different from normal refresh)?

If this doesn't work, please email us at support@gitbook.com so we can ask for more details and investigate. 

Thank you!
